src/demuxer.rs

```diff
@@ -176,12 +180,53 @@ fn track_entry_codec_id(t: &TrackEntry) -> Option<String> {
 
 fn track_entry_video_kind(t: &TrackEntry) -> Option<MediaKind> {
     // TODO: Validate that a track::video exists for track::type video before.
+
     if let Some(ref video) = t.video {
+        let mut format = None;
+
+        if let Some(ref colour) = video.colour {
+            let chroma = &[
+                Chromaton::new(0, 0, false, 8, 0, 0, 1),
+                Chromaton::new(
+                    colour.cb_subsampling_vert.unwrap_or_else(|| 0) as u8,
+                    colour.cb_subsampling_horz.unwrap_or_else(|| 0) as u8,
+                    false,
+                    0,
+                    8,
+                    0,
+                    1,
+                ),
+                Chromaton::new(
+                    colour.cb_subsampling_vert.unwrap_or_else(|| 0) as u8,
+                    colour.cb_subsampling_horz.unwrap_or_else(|| 0) as u8,
+                    false,
+                    0,
+                    8,
+                    0,
+                    1,
+                ),
+            ];
+
+            let f= Arc::new(Formaton::new(
+                ColorModel::CMYK, // where to check for other color scheme?
```

Comment:
why cmyk ?

Reply:
You should look at https://www.matroska.org/technical/specs/index.html#ColourSpace and map it as model.